python/seldon_core/batch_processor.py

```diff
@@ -17,6 +17,29 @@
 CHOICES_LOG_LEVEL = ["debug", "info", "warning", "error"]
 
 
+out_queue_empty_event = Event()
+
+counter_lock = Lock()
+read_counter = 0
+write_counter = 0
+
+
+def increment_read():
+    global read_counter
+    with counter_lock:
+        read_counter += 1
+
+
+def increment_write():
+    global write_counter
+    with counter_lock:
+        write_counter += 1
+
+
+def print_counters():
```

Comment:
remove

Reply:
it may be useful part of logging, right now they are very limited so at least having information how many lines were read and written out may be helpful

Reply:
removed